nessus_iam.tf

```diff
@@ -36,6 +36,15 @@ resource "aws_iam_role_policy_attachment" "cloudwatch_agent_policy_attachment_ne
   policy_arn = "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
 }
 
+# Attach the SSM Agent policy to this role as well
+resource "aws_iam_role_policy_attachment" "ssm_agent_policy_attachment_nessus" {
+  count    = lookup(var.operations_instance_counts, "nessus", 0) > 0 ? 1 : 0
+  provider = aws.provisionassessment
+
+  role       = aws_iam_role.nessus_instance_role[count.index].id
+  policy_arn = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+}
+
```

Comment:
I just merged the [PR that adds `gophish_iam.tf`](https://github.com/cisagov/cool-assessment-terraform/pull/45) - can you please add this sauce to that file as well?

Reply:
Will do!

Reply:
See commit 64bff80 and cisagov/pca-gophish-composition-packer#8.